ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -630,13 +630,13 @@ You should replace "${annotationProcessorPrefix}:${dependency}" with "annotation
     );
 
     // Add dependencies to result
-    dependencies.regular.forEach((d) => result.push(`implementation '${d}'`));
-    dependencies.files.forEach((d) => result.push(`implementation ${d}`));
+    dependencies.regular.forEach((d) => result.push(`api '${d}'`));
+    dependencies.files.forEach((d) => result.push(`api ${d}`));
     dependencies.raw.forEach((d) => {
       result.push(d);
     });
     dependencies.transitive.forEach((d) =>
-      result.push(`implementation ('${d}') { transitive = true }`),
+      result.push(`api ('${d}') { transitive = true }`),
```

Comment:
Mmmh.. While this might fix the problem with missing transitive dependencies, it is probably not the correct solution in terms of dependency structure (since it will be applied to _all_ dependencies) - Let's discuss tomorrow!